### PR TITLE
Add sorting to productList

### PR DESCRIPTION
**Pull request by Vomcat**

### CODE REVIEW

**Review by p-szalast — APPROVED** [review]

File: src/components/Product/ProductList.tsx [hunk 1]
```diff
@@ -44,18 +45,38 @@ const ProductList = () => {
...
-          name={item.name}
-          price={item.price}
-          imageURI={item.imageURI}
-        />
-      ))}
-    </ProductListWrapper>
+    <>
+      <ProductSearchBar changeHandler={onSelectSortHandler} />
+      <ProductListWrapper>
+        {items.map((item) => (
```
Comment (magnowak) [inline src/components/Product/ProductList.tsx]: could do:
`{items.map(({id, name, price, imageURI}) => (
<Item
key={id}
id={id}
name={name}
...
/>
))}`

**Review by magnowak — APPROVED** [review]

File: src/components/Product/ProductSearchBar.tsx [hunk 2]
```diff
@@ -41,7 +40,9 @@ const TextElement = styled.div`
   }
 `;
 
-const ProductSearchBar = () => {
+const ProductSearchBar = (props: ProductSearchBarProps) => {
```
Comment (magnowak) [inline src/components/Product/ProductSearchBar.tsx]: if this component no longer does any search functionality, change to ProductSortBar

File: src/components/Product/ProductList.tsx (hunk 1 above)
Comment (Vomcat) [reply src/components/Product/ProductList.tsx]: Changed

File: src/components/Product/ProductSearchBar.tsx (hunk 2 above)
Comment (Vomcat) [reply src/components/Product/ProductSearchBar.tsx]: Changed

**Pull request merged**